main/wallet_system.c

```diff
@@ -467,6 +484,86 @@ static void register_send() {
   ESP_ERROR_CHECK(esp_console_cmd_register(&send_cmd));
 }
 
+/* 'transactions' command */
+static struct {
+  struct arg_str *address;
+  struct arg_end *end;
+} get_transactions_args;
+
+static int fn_get_transactions(int argc, char **argv) {
+  retcode_t ret_code = RC_OK;
+  flex_trit_t tmp_hash[FLEX_TRIT_SIZE_243];
+
+  int nerrors = arg_parse(argc, argv, (void **)&get_transactions_args);
+  if (nerrors != 0) {
+    arg_print_errors(stderr, get_transactions_args.end, argv[0]);
+    return 1;
+  }
+
+  char const *address_ptr = get_transactions_args.address->sval[0];
+
+#if 0
```

Comment:
remove

Reply:
address pointer is requested by the following code, what  I need to remove?

Reply:
I meant the unused code, `#if 0`